controllers/virtualmachinegroup/virtualmachinegroup_controller_test.go

```diff
@@ -451,7 +466,7 @@ var _ = Describe(
 				}
 				intgFakeVMProvider.Unlock()
 
-				setVMUniqueID := func(vmKey types.NamespacedName, uniqueID string) {
+				setVMUniqueID = func(vmKey types.NamespacedName, uniqueID string) {
```

Comment:
nit: I saw other folks just define this helper func after `AfterEach`, before tests

Reply:
Yep, it should be outside BeforeEach/AfterEach. I didn't wanna move it to cause more diff lines for easier reviews. Updated in the latest commit to define all functions outside.